test_test_cases.R

```diff
@@ -0,0 +1,66 @@
+library(readr)
+library(yaml)
+
+test_cases_dir <- "~/Dropbox (2° Investing)/2° Investing Team/People/Jacob/p2020_test_cases"
+
+test_cases_csvs <- list.files(test_cases_dir, pattern = "\\.csv", full.names = TRUE)
+
+test_cases_output_dir <- "test_cases"
+
+unlink(test_cases_output_dir, recursive = TRUE)
```

Comment:
The directory `test_cases` should technically never exist and never be left around.... it is simply the export directory for the results of running the test_cases. I talked to Clare about this briefly... I can imagine someone wanting to save the test results from one run somewhere else even after changes to the repos have been made, but I think it's safest to wipe out anything in that directory (if it exists) when the tests begin running to guarantee there's no confusion about whether the results in `test_cases` are from the current/last test run or from a previous test run.

Reply:
Great. Sounds like a good use for `tempdir()` or equivalent from the smart package withr by Jim Hester.

Reply:
yeah, the problem is that whoever is running this test, they probably will want to inspect some of the results after the test runs in addition to reviewing the output in the console for errors and warnings, so accessing the folder after it runs should probably be easy-ish, which is not super true with `tempdir()`, otherwise that's totally what I would have preferred. I could open that directory automatically when the tests finish, but I don't want to get too fancy here.